context/src/main/java/com/c8y/ms/templates/context/service/EventService.java

```diff
@@ -0,0 +1,32 @@
+package com.c8y.ms.templates.context.service;
+
+import com.cumulocity.microservice.subscription.service.MicroserviceSubscriptionsService;
+import com.cumulocity.rest.representation.event.EventRepresentation;
+import com.cumulocity.sdk.client.event.EventApi;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+@Service
+public class EventService {
+
+    @Autowired
+    @Qualifier("userEventApi")
+    EventApi userEventApi;
+
+    @Autowired
+    EventApi tenantEventApi;
+
+    @Autowired
+    MicroserviceSubscriptionsService subscriptionsService;
```

Comment:
The MicroservceSubscriptionsService can be removed is not used at this service.

Reply:
removed with latest commit